app/src/main/java/ar/com/wolox/android/example/ui/auth/login/LoginFragment.kt

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.example.ui.auth.login
+
+import ar.com.wolox.android.R
+import ar.com.wolox.android.databinding.FragmentLoginBinding
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+
+class LoginFragment private constructor() : WolmoFragment<FragmentLoginBinding, LoginPresenter>(), LoginView {
```

Comment:
You don't need to set `private constructor()`. Just with `class LoginFragment : WolmoFragment...` it's ok!